compiler/env/RegionProfiler.hpp

```diff
@@ -89,7 +94,7 @@ class RegionProfiler
    size_t const _initialRegionSize;
    size_t const _initialSegmentProviderSize;
    TR::Compilation &_compilation;
-   const char * const _identifier;
+   char _identifier[512];
```

Comment:
Is this the size per region profilier? This is just to hold the description string right? If so 512 seems like a very large number given most of the current strings are "comp/opt", "comp/codegen", etc.

Reply:
Good point. I have cut it back to the original 256.